deploy/eck-stack/charts/eck-kibana/values.yaml

```diff
@@ -42,3 +42,69 @@ spec:
     # will be assumed.
     #
     # namespace: default
+
+ingress:
+  enabled: false
+  labels: {}
+  annotations: {}
+    # nginx.ingress.kubernetes.io/backend-protocol: "https"
+    # nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    # nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+    # cert-manager.io/cluster-issuer: my-issuer
+
+  # -- Defines which ingress controller will implement the resource
+  # -- uses class set as default by default
+  ingressClassName: ""
+
+  # -- Kibana hostname
+  hostname: "kibana.cluster.local"
+
+  service.port: 5601
+
+  # -- The path to Kibana
+  path: /
+
+  # -- Ingress path type. One of `Exact`, `Prefix` or `ImplementationSpecific`
+  pathType: Prefix
+
+  # -- Enable TLS configuration for the hostname defined at `server.ingress.hostname`
+  tls: false
+
+  # -- The list of additional hostnames to be covered by ingress record
+  # @default -- `[]` (See [values.yaml])
+  extraHosts: []
+    # - name: kibana.example.com
+    #   path: /
+
+  # -- Additional ingress paths
+  # @default -- `[]` (See [values.yaml])
+  ## Note: Supports use of custom Helm templates
+  extraPaths: []
+    # - path: /*
+    #   pathType: Prefix
+    #   backend:
+    #     service:
+    #       name: ssl-redirect
+    #       port:
+    #         name: use-annotation
+
+  # -- Additional ingress rules
+  # @default -- `[]` (See [values.yaml])
+  ## Note: Supports use of custom Helm templates
+  extraRules: []
+    # - http:
+    #     paths:
+    #     - path: /
+    #       pathType: Prefix
+    #       backend:
+    #         service:
+    #           name: ''
+    #           port:
+    #             name: ''
+
+  # -- Additional TLS configuration
+  # @default -- `[]` (See [values.yaml])
+  extraTls: []
+    # - hosts:
+    #   - kibana.example.com
+    #   secretName: your-certificate-name
```

Comment:
Sorry for the delays @jamesagarside in getting back to this. I'm assigned to this now and will follow it through to completion. I needed a better understanding of ingress as a whole, and how CSPs handle them and went down a bit of a rabbit hole working through getting ES and Kibana working in all cloud providers and opened #7941 as a discussion item. I wanted to discuss the differences between your implementation of the values, and what I had come up with https://github.com/elastic/cloud-on-k8s/pull/7941/files#diff-19d8910e9e220072251887cb93679712a3b08bf3dfc24151edaa6304aa72989aR79-R121.

I feel that this implementation in this PR may be overly complex for what our use case is, which is simply getting Kibana exposed to the public internet. If you could, compare the 2 implementations and let me know what you think of the suggestions from my PR. Specifically I'm wondering what the use case is for the `extra*` fields in the values file here in the context of Kibana.

Thanks, and looking forward to your input.

Reply:
@jamesagarside and I discussed this, and agreed to go ahead and go with what exists in #7941. We'll close this. Thanks much for kicking off this process to get ingress in place!